modules/ROOT/pages/configure-acm.adoc

```diff
@@ -311,14 +313,10 @@ In this section, you create the profile for members of your community. Users tha
 Next you need to give permissions to your members' profile so that they can access APIs, create client applications and manage their details. To do this:
 
 . Select *Edit* in the *ACM Member User* profile page.
-. In the *Standard Object Permissions* section:
-.. Enable *Read* permissions for:
-... *Documents*
+. In the *Standard Object Permissions* section, enable *Read* permissions for: *Documents*
```

Comment:
```suggestion
. In the *Standard Object Permissions* section, enable *Read* permissions for *Documents*.
```
GLOBALLY be sure that you are using bold only as specified in our style reference.
Changes *might* be forthcoming, but for now, we will work with the guidelines there.

Reply:
GLOBALLY avoid using colons within sentences.

Use vertical lists instead of narrative lists of more than a couple of items, and do not introduce those narrative lists with text ending in a colon.

Do not include colons at the end of UI labels in the UI labels when you use them in text.

I will stop marking colons; please review and fix as necessary.

Reply:
AND I will beef up the style ref!

Reply:
Good point about bold. This file definitely overuses bold in ways it should not, and I will include that in the fixes when we do a full edit on it.

I'll review the colons in this file and fix in this PR.